Jenkinsfile

```
@@ -0,0 +1,101 @@
+def kubernetes_config = """
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: ci
+    image: eclipseglsp/ci:0.0.4
+    tty: true
+    resources:
+      limits:
+        memory: "4Gi"
+        cpu: "2"
+      requests:
+        memory: "4Gi"
+        cpu: "2"
+    command:
+    - cat
+    volumeMounts:
+    - mountPath: "/home/jenkins"
+      name: "jenkins-home"
+      readOnly: false
+    - mountPath: "/.yarn"
+      name: "yarn-global"
+      readOnly: false
+  volumes:
+  - name: "jenkins-home"
+    emptyDir: {}
+  - name: "yarn-global"
+    emptyDir: {}
+"""
+pipeline {
+    agent {
+        kubernetes {
+            label 'glsp-agent-pod'
```

```suggestion
            label 'emfcloud-agent-pod'
```
Should use the emfcloud pod ;)